cluster/addons/pdcsi-driver/pd-csi.yaml

```diff
@@ -0,0 +1,188 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: csi-gce-pd-node
+  namespace: kube-system
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  selector:
+    matchLabels:
+      app: gcp-compute-persistent-disk-csi-driver
+  template:
+    metadata:
+      labels:
+        app: gcp-compute-persistent-disk-csi-driver
+    spec:
+      containers:
+      - args:
+        - --v=5
+        - --csi-address=/csi/csi.sock
+        - --kubelet-registration-path=/var/lib/kubelet/plugins/pd.csi.storage.gke.io/csi.sock
+        env:
+        - name: KUBE_NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.1.0
+        name: csi-driver-registrar
+        volumeMounts:
+        - mountPath: /csi
+          name: plugin-dir
+        - mountPath: /registration
+          name: registration-dir
+      - args:
+        - --v=5
+        - --endpoint=unix:/csi/csi.sock
+        - --run-controller-service=false
+        image: gke.gcr.io/gcp-compute-persistent-disk-csi-driver:v1.2.1-gke.0
```

Comment:
Maybe we should use the upstream cloud image, see the conversation I just started in our internal storage-dev chat.

Reply:
should we wait for the stable-master to change first and then change this with it

Reply:
hmm, good point.

How about we leave this version as you have here. It will be easier/better to get this in as-is, and have a small follow-up to change the image version, than to block this PR on the csi driver release.

Reply:
sounds good to me!

Reply:
We've already changed the CI to use the community images, we should use them here so we don't regress.

Reply:
@msau42 do you mean [here](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/testing-manifests/storage-csi/gce-pd/node_ds.yaml#L51)

Reply:
Yup

Reply:
done